week3/week3_day1.R

```diff
@@ -0,0 +1,238 @@
+
+#######################table reproduction
+library(tidyverse)
+library(scales)
+
+library(modelr)
+options(na.action = na.warn)
+
+library(broom)
+
+theme_set(theme_bw())
+options(repr.plot.width=4, repr.plot.height=3)
+
+
+body_model <- read.table("body.dat.txt", sep = "")
+
+
+#reference
+# https://stackoverflow.com/questions/10085806/extracting-specific-columns-from-a-data-frame
+library(dplyr)
+#okay so here is my 2 columns, weight and height
+my_table <- body_model %>% select(V23, V24)
+
+#reference https://www.marsja.se/how-to-rename-column-or-columns-in-r-with-dplyr/#:~:text=To%20rename%20a%20column%20in,dataframe%2C%20B%20%3D%20A)%20.
+my_table <- my_table %>% rename(weight = V23, height = V24)
+
+#there is a formula given from the website which is Weight (kg) = -105 + 1.018 Height (cm)
+#i don't think i can plug in constant numbers though, I think
+#i can only give the columns and let the computer do it itself
+form <- as.formula(weight ~ height)
+
+model <- lm(form, my_table)
+tidy(model)
+

```

Comment:
You can actually plug the formula syntax directly into lm and it works fine. The formula syntax though is good to know, since it can be reused throughout the script if you have it as a formula object.